app/components/balance-header/balance-header.tsx

```diff
@@ -88,19 +93,39 @@ type Props = {
 
 export const BalanceHeader: React.FC<Props> = ({ loading }) => {
   const isAuthed = useIsAuthed()
+  const { formatMoneyAmount } = useDisplayCurrency()
+  const { convertMoneyAmount } = usePriceConversion()
 
   // TODO: use suspense for this component with the apollo suspense hook (in beta)
   // so there is no need to pass loading from parent?
   const { data } = useBalanceHeaderQuery({ skip: !isAuthed })
-  const usdWalletDisplayBalance = isAuthed
-    ? data?.me?.defaultAccount?.usdWallet?.displayBalance ?? NaN
-    : 0
 
-  const btcWalletDisplayBalance = isAuthed
-    ? data?.me?.defaultAccount?.btcWallet?.displayBalance ?? NaN
-    : 0
+  let balanceInDisplayCurrency = "$0"
```

Comment:
Not in the scope of this PR, but there are several areas where we are making the assumption that the default display currency is "$". It would be better to define the default display currency in one place and the reference that config.